CMakeLists.txt

```diff
@@ -403,6 +403,7 @@ add_custom_target(check-integration
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
     COMMENT "Running integration tests"
     USES_TERMINAL
+    DEPENDS ${PROJECT_AWE_NAME}
```

Comment:
Should have more probably, similar to `check-themes`?
https://github.com/awesomeWM/awesome/pull/1967/files#diff-af3b638bc2a3e6c650974192a53c7291R413

Reply:
Yup, `generated_icons generate_awesomerc` definitely are needed by the integration tests.
I'd propose to just merge this one commit and if we stumble over more missing deps in the future, we will add them. (And perhaps refactor everything so that there is a target that depends on "everything needed" and where check-themes, check-integration etc just depend on)